test/server/wtr-config.test.js

```diff
@@ -0,0 +1,120 @@
+import { createConfig, getBrowsers, WTRConfig } from '../../src/server/wtr-config.js';
+import { createConfig as createConfigPublic, getBrowsers as getBrowsersPublic } from '../../src/index.js';
+import { expect } from 'chai';
+import sinon from 'sinon/pkg/sinon-esm.js';
+
+describe('createConfig()', () => {
```

Comment:
It might make sense to wrap all of these in a single "createWtrConfig" suite. That way as the tests are reported they'll all be grouped under that and it'll be easy to tell what functionality they belong to if one fails.

A single parent suite also lets you do common cleanup tasks like `sinon.restore`.